pkg/manager/server.go

```diff
@@ -21,41 +21,97 @@ import (
 	"errors"
 	"net"
 	"net/http"
+	"time"
 
 	"github.com/go-logr/logr"
+
+	crlog "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var (
+	_ Runnable               = &Server{}
```

Comment:
nit: I think the other invocation keeps you from allocating `Server`s

```
var (
    _ Runnable = (*Server)(nil)
    _ LeaderElectionRunnable = (*Server)(nil)
)
```

Reply:
Doesn't the compiler optimize out the allocation because of the `_` variable assignment?

Reply:
Either way, official docs use your example. I'll switch over. 

https://go.dev/doc/effective_go#blank_implements